spectacles/validators.py

```diff
@@ -43,9 +43,18 @@ class SqlValidator(Validator):
     """
 
     timeout = aiohttp.ClientTimeout(total=300)
+    MIN_LOOKER_VERSION = "6.22.12"
```

Comment:
Just wanting to confirm this is in fact the min version required, because #81 mentions `6.20.12`?

Reply:
Looks like the version in #81 is a typo (I've fixed it). However, we were told by Looker
that the API fix required here should be backported to 6.20, but I’m not
sure if that’s happened yet.

Anything after 6.22.12 should have the fix we’re checking for here, but if
the back port has occurred it should work on 6.20+

If you are able to get it working on pre-6.22.12, please let us know!